centreon/www/front_src/src/Dashboards/SingleInstancePage/Dashboard/AddEditWidget/WidgetProperties/Inputs/Resources/useResources.ts

```diff
@@ -390,7 +390,7 @@ const useResources = ({
 
   const getSearchField = (resourceType: string): string =>
     cond([
-      [equals('host'), always('h.name')],
+      [equals('host'), always('host.name')],
```

Comment:
why you changed the key to host.name 🤔 in fact that was corrected to h.name according to a bug 
https://centreon.atlassian.net/jira/software/c/projects/MON/issues/MON-142835?jql=project%20%3D%20%22MON%22%20ORDER%20BY%20created%20DESC

Reply:
@Noha-ElAbrouki Thanks for your return. I had a probleme with git and I needed to revert my branch. But I think I took the wrong commit to do this. So, I am going to delete this change from my PR.

Reply:
I put the file back as it was, it's ok now 😄